renderdoc/driver/gl/gl_enum.h

```diff
@@ -2428,6 +2428,7 @@ enum RDCGLenum
   eGL_ABGR_EXT = 0x8000,
   eGL_BGR_EXT = 0x80E0,
   eGL_BGRA_EXT = 0x80E1,
+  eGL_BGRA8_EXT = 0x93A1,
```

Comment:
Didn't realize this file is autogenerated.

BGRA8_EXT requires EXT_texture_storage and EXT_texture_format_BGRA8888. Should I figure out how to autogenerate the header or #define this inside gl_emulated.cpp?